verifier/verifier.go

```diff
@@ -721,11 +721,19 @@ func verifyRevocation(outcome *notation.VerificationOutcome, r revocation.Revoca
 		}
 	}
 
-	authenticSigningTime, err := outcome.EnvelopeContent.SignerInfo.AuthenticSigningTime()
-	if err != nil {
-		logger.Debugf("Not using authentic signing time due to error retrieving AuthenticSigningTime, err: %v", err)
-		authenticSigningTime = time.Time{}
+	var authenticSigningTime time.Time
+	var err error
+	if outcome.EnvelopeContent.SignerInfo.SignedAttributes.SigningScheme == signature.SigningSchemeX509SigningAuthority {
+		authenticSigningTime, err = outcome.EnvelopeContent.SignerInfo.AuthenticSigningTime()
+		if err != nil {
+			return &notation.ValidationResult{
+				Type:   trustpolicy.TypeRevocation,
+				Action: outcome.VerificationLevel.Enforcement[trustpolicy.TypeRevocation],
+				Error:  fmt.Errorf("unable to check revocation status, err: %s", err.Error()),
+			}
+		}
```

Comment:
This error looks wrong; for signing authority there is no revocation check for stimestamp certs as there is no timestamp signature. 
Also authentic signing time must have been checked before `verifyRevocation` call, so in what case this error will be thrown? [ref](https://github.com/notaryproject/notation-go/blob/b52583166f2b98c75899356aaf4964e98c261c48/verifier/verifier.go#L697)

Reply:
@priteshbandi The original logic is calling [`SignerInfo.AuthenticSigningTime`](https://github.com/notaryproject/notation-core-go/blob/f45197cbd53b9cbaf867cfe5152e3b8cf4ea9766/signature/types.go#L207). Since we upgraded notation-core-go, we need to update its usage in notation-go.

Since it's under `notary.x509.signingAuthority`, the only scenario that would trigger an error is `signerInfo.SignedAttributes.SigningTime.IsZero() == true`, i.e., there's no `signingTime` in the signature envelope. So yes, since we should already checked this before revocation, this error should never be triggered.

>This error looks wrong; for signing authority there is no revocation check for stimestamp certs as there is no timestamp signature.

Yes, this change is not related to timestamp as it's a concept under `notary.x509`.

Reply:
Since we know that this wold never happen, can we remove this check?

Reply:
Updated.